### ISSUE TITLE
Multiple events assigned to one callback

### ISSUE THREAD

**Issue by shshaw** [kenwheeler/cash#130]
Space separated events to assign multiple events to one callback.

```
$element.on('mouseenter focusin',callback);

// Equivalent to:
$element.on('mouseenter',callback);
$element.on('focusin',callback);
```

**Comment by fabiospampinato** [comment]
I've implemented this in [my fork](https://github.com/fabiospampinato/cash).

**Comment by fabiospampinato** [comment]
Fixed in v2.1.2.
